docs/api/api.md

```diff
@@ -17,6 +17,10 @@ zkSync Era supports the standard [Ethereum JSON-RPC API](https://ethereum.org/en
 
 ## RPC endpoint URLs
 
+:::warning Rate Limits
+
+We apply rate limiting to both HTTPs and Websocket APIs. The limits are generally permissive (currently 10s to 100s RPS per client), but please contact us if you face any issues in that regard.
```

Comment:
Please either capitalize all of `HTTPS` or leave as `Https`. `HTTPs` makes it sound like there are multiple `HTTP` 😄